go.mod

```diff
@@ -34,6 +41,7 @@ require (
 	github.com/jessevdk/go-flags v1.4.1-0.20200711081900-c17162fe8fd7
 	github.com/jrick/logrotate v1.0.0
 	github.com/lib/pq v1.2.0
+	github.com/lightninglabs/neutrino v0.12.1
```

Comment:
Patch release v0.12.3 tagged a couple weeks ago. https://github.com/lightninglabs/neutrino/compare/v0.12.1...v0.12.3
Just one fix to rebroadcasting code it appears.